source/Nevermore.Tests/QueryBuilderFixture/QueryBuilderFixture.ShouldGenerateSelectForComplicatedSubqueryJoin.approved.txt

```diff
@@ -1,4 +1,3 @@
-SELECT t1.* FROM (SELECT * FROM dbo.[Orders]) t1
-INNER JOIN (SELECT * FROM dbo.[Customers] WHERE (IsActive = 1)) t2 ON t1.[CustomerId] = t2.[Id] AND t1.[Owner] = t2.[Owner]
-INNER JOIN (SELECT * FROM dbo.[Accounts] WITH (UPDLOCK)) t3 ON t1.[AccountId] = t3.[Id]
-ORDER BY t1.[Id]
\ No newline at end of file
+SELECT t2.* FROM dbo.[Orders] t2
+INNER JOIN (SELECT * FROM dbo.[Customers] WHERE (IsActive = 1) ORDER BY [Id]) t1 ON t2.[CustomerId] = t1.[Id] AND t2.[Owner] = t1.[Owner]
```

Comment:
In the test method, this subquery is generated by:

```
            var customers = CreateQueryBuilder<IDocument>("Customers")
                .Where("IsActive = 1")
                .OrderBy("Id");
```

So the order by is only added to this subquery because the consumer explicitly added it there. I can rename it to something other than `Id` though to make it clearer that it is not coming from the default order by clause.

Reply:
Ah right, nevermind then

Reply:
I have changed the field name